TODO.txt

```diff
@@ -21,14 +21,14 @@ Version 0.17
 Version 0.18
 ------------
 
+* Remove deprecated function ``guess_spatial_dimensions`` in ``skimage.color.__init__``.
```

Comment:
Should it be 0.17?

Reply:
No, two versions from the version in which this is released, which will be 0.16.